auth/authenticate-middleware.js

```diff
@@ -1,8 +1,31 @@
-/* 
-  complete the middleware code to check if the user is logged in
-  before granting access to the next middleware/route handler
-*/
+const jwt = require('jsonwebtoken');
+const secrets = require('../config/secrets');
+
+const Users = require('./auth-model');
 
 module.exports = (req, res, next) => {
-  res.status(401).json({ you: 'shall not pass!' });
-};
+  const tokenHeader = req.headers.authorization;
+
+  if(tokenHeader) {
+    const tokenHeader = req.headers.authorization;
+
+    if(tokenHeader) {
+      const tokenStrings = tokenHeader.split(" ");
```

Comment:
these next several lines only work if when you test in postman, you add BEARER to your token....if you don't have BEARER, you don't need these next few lines where you split it. The first argument in your verify can just be req.headers.authorization.